package.json

```diff
@@ -21,12 +21,12 @@
     "dist/**/*"
   ],
   "engines": {
-    "node": ">=12.13.0",
-    "npm": ">=6.12.0"
+    "node": ">=14.21.3",
+    "npm": ">=6.14.18"
   },
   "scripts": {
     "prepare": "npm run build",
-    "build": "tsc",
```

Comment:
Ensure we clean out the `dist/` directory of transpiled files to be published first before building.